app/controllers/volunteers_controller.rb

```diff
@@ -56,7 +56,7 @@ def destroy
 
   def seeking_clients
     authorize Volunteer
-    @q = Volunteer.seeking_clients.ransack(params[:q])
+    @q = specific_policy_scope(:seeking_clients).ransack(params[:q])
```

Comment:
Is this the same as `policy_scope(Volunteer).seeking_clients`? Because that would be a lot easier to follow ;-)